app/models/user.rb

```diff
@@ -1,8 +1,15 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
The application should ask for confirmation of email. Kindly add `:confirmable` attribute to the devise, so as when a new user registers, it asks for an email confirmation.